.github/workflows/ci.yml

```diff
@@ -18,9 +18,8 @@ jobs:
         uses: actions/setup-node@v2
         with:
           node-version: 12.x
-          cache: yarn
```

Comment:
I think we can keep this around with `cache: npm` if you so desire!
 
https://github.com/actions/setup-node